actindo-grid.html

```diff
@@ -1252,50 +1270,99 @@
                 return column;
             }
 
+
             /**
-             * called when the columnConfiguration changes. Process the columns and copy them to the internal
-             * _columns property.
-             *
-             * @param {{base: []}} columnConfiguration
+             * call when column configuration change
              * @private
              */
-            _consolidateColumns(columnConfiguration) {
-                let configuration = columnConfiguration.base;
+            _onColumnConfigurationChanged(columns)
+            {
+                const column = columns.value;
+
+                if (typeof columns.value !== 'object')
+                {
+                    return;
+                }
 
-                let columns = [];
+                // Do not reload if the column is hidden or not optional
+                if (!column.optional || column.hidden)
+                {
+                    return;
+                }
 
-                let filterColumns = [] ;
+                if (this._serializeOptionals.includes(column.field))
+                {
+                    return;
+                }
 
-                this._identifyingColumn = null;
+                this.push('_serializeOptionals', column.field);
 
-                for(let i = 0, c = configuration.length; i < c; i++)
+                this._loadDebounced(
```

Comment:
You don't need this anymore if you add `_serializeOptionals` to the `_onRemoteChanged` listener above